### PR TITLE
Feature/qccollect cli

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- qccollect is now part of the MIP CLI and not a separate script

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
@jemten Sorry, this took some more work than I thought. I think it is ready now

File: lib/MIP/Program/Mip.pm [hunk 1]
```diff
@@ -23,19 +23,148 @@ BEGIN {
...
+            strict_type => 1,
+            store       => \$append_stderr_info
+        },
+        skip_evaluation => {
+            default     => 0,
+            allow       => [ 0, 1 ],
+            strict_type => 1,
+            store       => \$skip_evaluation
+        },
+    };
```
Comment (jemten) [inline lib/MIP/Program/Mip.pm]: Order AB for the parameters above

File: lib/MIP/Program/Mip.pm [hunk 2]
```diff
@@ -23,19 +23,148 @@ BEGIN {
...
+        push @commands, q{--outfile} . $SPACE . $outfile_path;
+    }
+
+    push @commands,
+      unix_standard_streams(
+        {
+            stdoutfile_path        => $stdoutfile_path,
+            stderrfile_path        => $stderrfile_path,
+            stderrfile_path_append => $stderrfile_path_append,
+        }
```
Comment (jemten) [inline lib/MIP/Program/Mip.pm]: order AB

File: lib/MIP/Program/Mip.pm [hunk 3]
```diff
@@ -101,7 +230,7 @@ my $pli_values_file_path;
             strict_type => 1,
             store       => \$per_gene,
         },
-pli_values_file_path => {
+        pli_values_file_path => {
             strict_type => 1,
             store       => \$pli_values_file_path,
         },
```
Comment (jemten) [inline lib/MIP/Program/Mip.pm]: Order AB

**Pull request merged**